processor/apmprocessor/span.go

```diff
@@ -35,7 +36,13 @@ func (sp *spanProcessor) processTraces(_ context.Context, td ptrace.Traces) (ptr
 			scopeSpan := rs.ScopeSpans().At(j)
 			for k := 0; k < scopeSpan.Spans().Len(); k++ {
 				span := scopeSpan.Spans().At(k)
-				if parsedTable, parsed := sp.sqlparser.ParseDbTableFromSpan(span); parsed {
+				if span.Kind() == ptrace.SpanKindServer || span.Kind() == ptrace.SpanKindConsumer {
+					transactionName, transactionType := apmconnector.GetTransactionMetricName(span)
+					if transactionType != apmconnector.NullTransactionType {
+						span.Attributes().PutStr("transaction.type", transactionType.AsString())
+						span.Attributes().PutStr("transaction.name", transactionName)
```

Comment:
To add to our ongoing conversation about attribute and metric names, we should consider keeping our attribute namespaces consistent with our metric names

`apm.service.transaction.type` and `apm.service.transaction.name`

Reply:
Ugh.  Yeah, something like that is probably the right thing to do.  Let's keep discussing this.